src/frontend/komponenter/Fagsak/Dokumentutsending/KanSøke/Dokumentvelger.tsx

```diff
@@ -1,44 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import styled from 'styled-components';
 
-import { Label } from 'nav-frontend-skjema';
-
-import { FamilieReactSelect } from '@navikt/familie-form-elements';
+import { Combobox } from '@navikt/ds-react/esm/form/combobox';
 
 import { useDokumentutsending } from '../../../../context/DokumentutsendingContext';
-import type { ISelectOptionMedBrevtekst } from '../../../Felleskomponenter/Hendelsesoversikt/BrevModul/typer';
-import {
-    leggTilValuePåOption,
-    opplysningsdokumenter,
-} from '../../../Felleskomponenter/Hendelsesoversikt/BrevModul/typer';
-
-const LabelOgEtikett = styled.div`
-    display: flex;
-    justify-content: space-between;
+import { opplysningsdokumenter } from '../../../Felleskomponenter/Hendelsesoversikt/BrevModul/typer';
+
+const Container = styled.div`
+    margin-bottom: 1rem;
 `;
+
 export const Dokumentvelger = () => {
     const { skjema } = useDokumentutsending();
+    const [value, setValue] = useState('');
```

Comment:
Trenger vi både denne og `onToggleSelected`? Ser ikke helt hvor vi bruker value etter at den blir satt

Reply:
value er ikke verdiene som er valgt, men teksten som står i inputfeltet. Dvs teksten saksbehandler skriver inn for å kunne søke blant valgene. Men det funker nok fint å ha den ukontrollert, jeg kan teste

Reply:
Testet litt nå. Må sette value på denne måten hvis jeg skal importere NavBaseSkjemaProps på den måten det skjer nå. Dette er fordi disse propsene har et verdi-felt som er verdien til skjema-feltet (altså en liste av strenger). Comboboxen forventer å få en streng som value og ikke en liste. Så tenker det er to alternativer her:
- Beholde value kontrollert sånn som PRen viser akkurat nå
- Fjerne value og onChange, og også hvordan vi henter inn feil fra NavBaseSkjemaProps

Noen tanker om hva som er best løsning?

Reply:
Hvis vi bare bruker feilmeldingene fra `hentNavBaseSkjemaProps` så høres det ikke så dumt ut å bare ekstrahere dem og sende dem videre til Combobox 🤔

Reply:
Enig! Fikser det 😄